### PR TITLE
Fix broken images and paths

### PR DESCRIPTION
**Pull request by AE-MS**
Some of the documentation I previously committed had broken images and links (due to it being sourced elsewhere).

* I have fixed up the images
* I have removed the links to source files since, while helpful, seem like something that is brittle and likely to break over time since people aren't going think of them as possibly being used as examples from this document. Anyone reading this document should know how to find the source file locations if needed.

### CODE REVIEW

**Comment by erinha** [comment]
In architecture.md, I noticed from the previous PR a typo:

> Capabilities can be feature or content-based, such a `mail`, `calendar`, `chat`, `dialog`, `authentication`, etc.,...

Should be "such as `mail`..."

**Comment by erinha** [comment]
Typo in architecture.md:

> It **strongly** discouraged that private capabilities be used for this purpose.

Missing "is" following "It".

**Comment by erinha** [comment]
Note about recent change in architecture.md:

A language identifier was added to the code blocks in the "Adding an API that utilizes version checks for compatibility" section, but it was added as "json" where the code blocks are actually "TypeScript".

File: packages/teams-js/devdocs/architecture.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ This option should only be used for work that meets ALL of the below requirement
 
 Here are the steps for adding an API that utilizes version checks (e.g. `if (!isCurrentSDKVersionAtLeast(captureImageMobileSupportVersion)...`):
 
-1. Add the API as a new capability or subcapability rather than adding to an existing capability. Please look at other capabilities such as [calendar.ts](packages/teams-js/src/public/calendar.ts) for examples of how to structure a capability. There must be an isSupported() function with every capability which is a simple boolean check for seeing if `runtime.supports` contains the capability.
+1. Add the API as a new capability or subcapability rather than adding to an existing capability. Please look at other capabilities such as `calendar.ts` for examples of how to structure a capability. There must be an isSupported() function with every capability which is a simple boolean check for seeing if `runtime.supports` contains the capability.
 
 e.g.
```
Comment (erinha) [inline packages/teams-js/devdocs/architecture.md:46]: Next line in TypeScript not json.

**Review by erinha — CHANGES_REQUESTED** [review]
Thanks for making this change! Comments regarding a couple typos from the previous PR touching these files.

File: packages/teams-js/devdocs/architecture.md (hunk 1 above)
Comment (erinha) [reply packages/teams-js/devdocs/architecture.md:46]: There are several code blocks in here that are mistyped as json and should be TypeScript (GitHub is not letting me highlight the line numbers 👎 ). Lines 57 and 76.

Comment (erinha) [reply packages/teams-js/devdocs/architecture.md:46]: They could be javascript vs TypeScript, I'm not sure if it matters.

**Review by erinha — CHANGES_REQUESTED** [review]
Two more lines that are json but should be javascript/TypeScript.

**Review by erinha — APPROVED** [review]
🚀

**Comment by erinha** [comment]
A change file is required, since the devdocs folder is under the packages/teams-js directory (which is what beachball looks at).

**Review by erinha — APPROVED** [review]
Beautiful change file comment!

**Review by lakhveerk — APPROVED** [review]
lgtm

**Review by MarcinJuraszek — APPROVED** [review]

**Pull request merged**